sherpa/astro/ui/tests/test_astro_ui_unit.py

```diff
@@ -431,20 +431,8 @@ def test_save_data_data1d_no_clobber(tmp_path):
 def test_save_data_datapha_no_clobber(tmp_path):
     """save_data: does clobber=False work? DataPHA"""
 
-    # This import requires an I/O backend hence it is done here
-    #
-    from sherpa.astro.io.meta import Meta
-
     ui.load_arrays(1, [1, 2, 3], [5, 4, 3], ui.DataPHA)
-
-    # The code requires DataPHA to have a "valid" header so
-    # fake one. Ideally we would not require it.
-    #
-    hdr = Meta()
-    ui.get_data().header = hdr
-
     out = tmp_path / "data.dat"
```

Comment:
dang it - it's not easy to be able to cheer on removed lines from this interface!